cockatrice/src/window_main.cpp

```diff
@@ -257,6 +257,8 @@ void MainWindow::actSettings()
 
 void MainWindow::actExit()
 {
+    if (QMessageBox::question(this, tr("Close Cockatrice"), tr("Are you sure you want to close Cockatrice?"), QMessageBox::Yes | QMessageBox::No, QMessageBox::No) != QMessageBox::Yes)
```

Comment:
I now get the Are you sure you want to close Cockatrice?"" twice when closing from the menu under osx.